Source/bmalloc/libpas/src/libpas/pas_large_sharing_pool.h

```diff
@@ -51,9 +51,9 @@ typedef struct pas_physical_memory_transaction pas_physical_memory_transaction;
 struct pas_large_sharing_node {
     pas_red_black_tree_node tree_node;
     
-    pas_commit_mode is_committed : 1;
-    pas_physical_memory_synchronization_style synchronization_style : 1;
-    pas_mmap_capability mmap_capability : 1;
+    bool is_committed : 1;
+    bool synchronization_style : 1;
+    bool mmap_capability : 1;
```

Comment:
This change does not look good. I think you should annotate this is non ms struct in clang-cl

Reply:
Updated to make this change Windows only with some extra comments. gcc_struct isn't available on clang-cl, unfortunately so I think this is about as good as it can get until C23 unlocks typed enumerations.